code/modules/atmospherics/FEA_gas_mixture.dm

```diff
@@ -371,11 +370,10 @@ What are the archived variables for?
 			var/datum/gas/corresponding = src.get_or_add_trace_gas_by_type(trace_gas.type)
 			corresponding.moles -= trace_gas.moles
 
-	return TRUE
-
 /// Checks if us and the sharer have a low enough delta and our combination's delta is small enough that group processing can be preserved.
-/// Returns: 0 if the self-check failed then -1 if sharer-check failed then 1 if both checks pass.
+/// Returns: 0 if the self-check failed, -1 if sharer-check failed, 1 if both checks pass.
 /datum/gas_mixture/proc/check_gas_mixture(datum/gas_mixture/sharer)
+	. = 1
```

Comment:
i confess to being unsure what this does

Reply:
also should it be TRUE instead of 1?

Reply:
i didnt like the return true at the end, albeit it doesnt make much sense now. also i cant use true since this function has 3 states, not 2.

Reply:
if its got three states maybe use defines instead of magic numbers

Reply:
idk where to put the defines.
atmos.dm?

Reply:
in atmos.dm or somewhere in _std/defines folder i suppose